### ISSUE TITLE
Colour for track features

### ISSUE THREAD

**Issue by carlosribas** [igvteam/igv.js#1643]
Hi,

I'm using the color property but I can still see some tracks with the default colour. I'm wondering if this is expected or if it's a bug. This is the code to test:

```
<html>
<body>

<div id="igvDiv"></div>
<script type="module">
    import igv from "https://cdn.jsdelivr.net/npm/igv@2.15.7/dist/igv.esm.js"

    var options =
        {
            genome: "hg38",
            locus: "X:73,819,307-73,856,333",
            tracks: [
                {
                    type: 'annotation',
                    format: 'gff3',
                    url: 'https://ftp.ebi.ac.uk/pub/databases/RNAcentral/.ensembl/homo_sapiens.GRCh38.sorted.gff3.gz',
                    indexURL: "https://ftp.ebi.ac.uk/pub/databases/RNAcentral/.ensembl/homo_sapiens.GRCh38.sorted.gff3.gz.tbi",
                    height: 600,
                    color: "red"
                }
            ]
        }

    var igvDiv = document.getElementById("igvDiv")

    igv.createBrowser(igvDiv, options)
        .then(function (browser) {
        })

</script>
</body>
</html>
```

output:

<img width="1502" alt="Screenshot 2023-05-17 at 17 02 57" src="https://github.com/igvteam/igv.js/assets/8561563/322f5a73-90d4-4aee-8431-d4c6296c4a51">

**Comment by jrobinso** [comment]
I will look into it.

**Comment by jrobinso** [comment]
This is a bug, you can work around it by also setting ```altColor```, which applies to negative strand features.  If altColor is not explicitly set it should default to color,  it is not doing that thus the bug.

**Comment by jrobinso** [comment]
This should be fixed with release 2.15.8

**Comment by carlosribas** [comment]
That's super! I can see that 2.15.8 fixed the issue, thank you!
